frontend/ui/widget/touchmenu.lua

```diff
@@ -594,16 +594,26 @@ function TouchMenu:updateItems()
     table.insert(self.item_group, self.footer_top_margin)
     table.insert(self.item_group, self.footer)
     -- @translators %1 is the current page. %2 is the total number of pages. In some languages a good translation might need to reverse this order, for instance: "Total %2, page %1".
-    self.page_info_text.text = util.template(_("Page %1 of %2"), self.page, self.page_num)
+    -- don't show page x of x if there is only one page
```

Comment:
Did you check what xgettext thinks of this? You might have to reverse the order. Also comments like to be proper sentences.

```suggestion
    -- Don't show page x of x if there is only one page.
```

However… I'd say this isn't really the kind of code that requires much comment.